backend/src/models/student.js

```diff
@@ -0,0 +1,36 @@
+const { Model } = require('objection')
+const tableName = 'students'
+
+class Student extends Model {
```

Comment:
I think we should store a relation mapping to link Student and Status: https://vincit.github.io/objection.js/api/model/static-properties.html#static-relationmappings. There is a `HasOneRelation` that looks promising.

Reply:
This is very helpful! Thank you! 

After using relation mapping in the objection model, should we still define the database schema in the migrations file? 
e.g. in migrations file for status table:

exports.up = (knex) => {
  return knex.schema.createTable(tableName, (table) => {
    table.increments('id').primary()
    table.text('id').index().**references('StatusID').inTable('students')**
  })
}

Reply:
I think we would define the reference in the students migration:
```js
...
return knex.schema.createTable(tableName, table => {
  // ...
  table.integer('StatusID').references('id').inTable('status')
  // ...
})
```

Btw, to create code snippets on Github:

\```js
Your code here
\```

Reply:
Ah... you are right, thank you 👍